src/test/scripts/helm_install.sh

```diff
@@ -47,6 +47,12 @@ helm install -n "${TARGET_NAMESPACE}" --wait \
    --version "$POSTGRES_CHART_VERSION" \
    bitnami/postgresql > $LOG_DOWNLOAD_DIR/helm_install_log.txt
 
+if [ -n "$2" ]
+then
+  kubectl cp $2 $TARGET_NAMESPACE/$POSTGRES_RELEASE_NAME-0:/tmp/db-init-script.sql
+  kubectl exec --stdin --tty $POSTGRES_RELEASE_NAME-0 -- /bin/bash -c "psql postgresql://$PRODUCT_NAME:$PRODUCT_NAME@localhost:5432/$PRODUCT_NAME -f /tmp/db-init-script.sql"
```

Comment:
do we need `-tstdin --tty`?

Reply:
Apparently not, plus it avoids line wrapping:
```
 ~ $ kubectl exec --stdin --tty kcichy-jira-pgsql-0 -- /bin/bash -c 'PGPASSWORD="jira" psql --host 127.0.0.1 -U jira -d jira -p 5432 -f /tmp/t.sql'
 id | pkey | issuenum | project | reporter | assignee | creator | issuetype | su
mmary | description | environment | priority | resolution | issuestatus | create
d | updated | duedate | resolutiondate | votes | watches | timeoriginalestimate
| timeestimate | timespent | workflow_id | security | fixfor | component | archi
ved | archivedby | archiveddate
----+------+----------+---------+----------+----------+---------+-----------+---
------+-------------+-------------+----------+------------+-------------+-------
--+---------+---------+----------------+-------+---------+----------------------
+--------------+-----------+-------------+----------+--------+-----------+------
----+------------+--------------
(0 rows)

 ~ $ kubectl exec kcichy-jira-pgsql-0 -- /bin/bash -c 'PGPASSWORD="jira" psql --host 127.0.0.1 -U jira -d jira -p 5432 -f /tmp/t.sql'
 id | pkey | issuenum | project | reporter | assignee | creator | issuetype | summary | description | environment | priority | resolution | issuestatus | created | updated | duedate | resolutiondate | votes | watches | timeoriginalestimate | timeestimate | timespent | workflow_id | security | fixfor | component | archived | archivedby | archiveddate
----+------+----------+---------+----------+----------+---------+-----------+---------+-------------+-------------+----------+------------+-------------+---------+---------+---------+----------------+-------+---------+----------------------+--------------+-----------+-------------+----------+--------+-----------+----------+------------+--------------
(0 rows)
```